blazor/diagram/getting-started/deploy-maui-windows.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: post
+title: Deploy Diagram Component in Blazor MAUI in windows | Syncfusion
+description: Learn how to getting started with Diagram control in Blazor MAUI application
```

Comment:
description: Learn how to get started with Diagram control in the Blazor MAUI application.